infra/gcp/bash/lib.sh

```diff
@@ -389,34 +392,49 @@ function empower_group_to_admin_artifact_auditor() {
         --role="roles/iam.serviceAccountUser"
 }
 
+# Grant write privileges to the file promoter bot
+# $1: The GCP project
+# $2: The bucket (e.g. gs://bucket-name)
+function empower_file_promoter() {
+    if [ $# -lt 2 ] || [ -z "$1" ] || [ -z "$2" ]; then
+        echo "empower_file_promoter(project, bucket) requires 2 arguments" >&2
+        return 1
+    fi
+    local project="$1"
+    local bucket="$2"
+    local acct
+    acct=$(svc_acct_email "${project}" "${FILE_PROMOTER_SVCACCT}")
+
+    ensure_service_account "${project}" "${FILE_PROMOTER_SVCACCT}" "artifact promoter"
+
+    # TODO(kpromo): Determine if admin access is required here.
+    #               ref: https://github.com/kubernetes-sigs/k8s-container-image-promoter/issues/413
+    empower_svcacct_to_write_gcs "${acct}" "${bucket}"
```

Comment:
Typo I found while running
```suggestion
    empower_svcacct_to_write_gcs_bucket "${acct}" "${bucket}"
```

Reply:
Fixed. Thanks for catching that!